pages/index.js

```diff
@@ -160,6 +161,14 @@ export default function Home({
     },
   ];
 
+  const featureRefs = featuresDetails.map(() => {
+    return useElementOnScreen({
+      root: null,
+      rootMargin: "0px",
+      threshold: 0.4,
+    });
+  });
```

Comment:
I think this can be rewritten to something like this...?

```suggestion
  const featureRefs = featuresDetails.map(() => useElementOnScreen({
      root: null,
      rootMargin: "0px",
      threshold: 0.4,
    }));
```

Reply:
Oops :) silly mistake. Fixed!!

Reply:
Thanks